README.md

```diff
@@ -83,26 +83,93 @@ npx task-master init
 ```
 
 This will prompt you for project details and set up a new project with the necessary files and structure.
+You can also specify brand rules to apply during initialization:
+
+```bash
+task-master init --rules cursor,windsurf
+```
 
 #### Common Commands
 
 ```bash
 # Initialize a new project
 task-master init
 
+# Initialize a new project applying specific brand rules
```

Comment:
lets not call them brand rules externally, just internally, so lets just call them rules